packages/desktop/components/popups/ActivityDetailsPopup.svelte

```diff
@@ -101,60 +101,62 @@
 </script>
 
 <activity-details-popup class="w-full h-full space-y-6 flex flex-auto flex-col shrink-0">
-    <div class="flex flex-col">
-        <Text type={TextType.h3} fontWeight={FontWeight.semibold} classes="text-left">
-            {localize('popups.transactionDetails.title')}
-        </Text>
-        {#if explorerUrl && activity?.transactionId}
-            <button
-                class="action w-max flex justify-start text-center font-medium text-14 text-blue-500"
-                on:click={onExplorerClick}
-            >
-                {localize('general.viewOnExplorer')}
-            </button>
-        {:else if activity?.transactionId}
-            <button
-                class="action w-fit flex justify-start text-center font-medium text-14 text-blue-500"
-                on:click={onTransactionIdClick}
-            >
-                {truncateString(activity.transactionId, 12, 12)}
-            </button>
+    {#if activity}
```

Comment:
why is this necessary? I mean, if we have activity details, then we have an activity, right? or maybe i'm misunderstood... :pray: :see_no_evil: